content/components/action-list.mdx

```diff
@@ -199,7 +199,7 @@ Action list items can be selected. Single selections are represented with a [`ch
 
 <Grid gridTemplateColumns={['1fr', null, null, null, '1fr 2fr']} gridGap={4}>
     <div>
-        <Link href="/components/action-menu">
+        <Link href="https://primer.style/components/ActionMenu">
```

Comment:
I don't think the `primer.style/` is necessary for these links.
```suggestion
        <Link href="/react/ActionMenu">
```

Reply:
Updated that @colebemis. Unfortunately still doesn't seem to work locally or on the preview link. I think it's because the React ones are on another site? 

That's why I was worried that `<Link />` wouldn't work either way as it wouldn't be on the same router? I could be super wrong.

Is there another way to test this? I guess if we can't test it before merging we could merge it anyway since it's broken anyway and smoke test it after it's deployed?